src/nimblepkg/semver.nim

```diff
@@ -0,0 +1,73 @@
+import algorithm, sequtils, strutils
```

Comment:
Nimble already has a version parsing module, could you re-use it instead of introducing this new one? (https://github.com/nim-lang/nimble/blob/master/src/nimblepkg/version.nim)

Reply:
Just pushed a commit that should address this.